### PR TITLE
Fix: Fixed null warnings in the details layout

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- Fixed null warnings in the details layout

### CODE REVIEW

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs [hunk 1]
```diff
@@ -136,7 +135,7 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
 
 			var parameters = (NavigationArguments)eventArgs.Parameter;
 			if (parameters.IsLayoutSwitch)
-				ReloadItemIconsAsync();
+				await ReloadItemIconsAsync();
```
Comment (hishitetsu) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:138]: Shouldn't we leave it as fire and forget?

Comment (yaira2) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:138]: Is there any downside to using `await`? If we remove it, should we suppress the warning and add a comment?

Comment (hishitetsu) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:138]: Await will cause the process to pause there, and no further processing will take place until icons are reloaded. If it is working fine now, this modification may lead to performance degradation.
If you just want to get rid of the warning, you can write the following:
```csharp
_ = ReloadItemIconsAsync();
```

Comment (yaira2) [reply src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs]: 👍

**Review by hishitetsu — APPROVED** [review]
LGTM

**Pull request merged**